BlueprintUI/Sources/Element/LayoutStorage.swift

```diff
@@ -72,6 +73,7 @@ extension LayoutStorage: LegacyContentStorage {
             )
 
             let resultNode = LayoutResultNode(
+                identifier: .identifier(for: currentChild.element, key: nil, count: childCount),
```

Comment:
Do we need to pass the `key` in here from the child? (I am not 100% sure but maybe?)